sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -28,10 +28,10 @@
 /**
  * A column vector backed by Apache Arrow.
  */
-public final class ArrowColumnVector extends ColumnVector {
+public class ArrowColumnVector extends ColumnVector {
```

Comment:
Shall we mark this class as `DeveloperApi`?

Reply:
Thanks, added